templates/zoom-parts/username-section.tpl

```diff
@@ -9,6 +9,7 @@
   <div class="linkWrapSection">
     <a id="UsernameUserPage" class="btn btn-small" target="_blank" href="https://en.wikipedia.org/wiki/User:{$usernamerawunicode|escape:'url'}" onMouseUp="$('#UsernameUserPage').addClass('btn-visited');">User page</a>
     <a id="UsernameCreationLog" class="btn btn-small" target="_blank" href="https://en.wikipedia.org/w/index.php?title=Special:Log&amp;type=newusers&amp;user=&amp;page={$usernamerawunicode|escape:'url'}" onMouseUp="$('#UsernameCreationLog').addClass('btn-visited');">Creation log</a>
+    <a id="UsernameGlobalRenameLog" class="btn btn-small" target="_blank" href="https://meta.wikimedia.org/w/index.php?title=Special%3ALog&type=gblrename&oldname={$usernamerawunicode|escape:'url'}" onMouseUp="$('#UsernameGlobalRenameLog').addClass('btn-visited');">Global Rename log</a>
```

Comment:
The ampersands ('&') at the start and end of `&type=gblrename&` in the URL (also conveniently highlighted in red in this diff) need to be replaced with HTML entities (i.e., `&amp;`) because raisins.  See the creation log button as an example.